### PR TITLE
Fixes bug 1260747 - Added a Correlations tab to Signature report.

### PR DESCRIPTION
**Pull request by adngdb**

### CODE REVIEW

File: webapp-django/crashstats/signature/jinja2/signature/signature_correlations.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<div id="correlations-wrapper" data-correlations-source="{{ url('crashstats:correlations_json') }}">
```
Comment (adngdb) [inline webapp-django/crashstats/signature/jinja2/signature/signature_correlations.html]: This file is mostly a copy of https://github.com/mozilla/socorro/blob/master/webapp-django/crashstats/crashstats/jinja2/crashstats/partials/correlations.html with a few minor adjustments.

File: webapp-django/crashstats/signature/static/signature/js/signature_tab_correlations.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
+/*global SignatureReport: true, Accordion: true, socSortCorrelation:true */
```
Comment (adngdb) [inline webapp-django/crashstats/signature/static/signature/js/signature_tab_correlations.js]: This file is mostly a copy of https://github.com/mozilla/socorro/blob/master/webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report_list_correlations.js with some adjustments to make it work in the context of Signature Report.

**Comment by peterbe** [comment]
This happens:
```
crashstats/js/lib/accordions.js                      in signature_report     already in report_list
REPEATED crashstats/js/lib/accordions.js
FOUND IN ['report_list', 'signature_report']
ALWAYS TOGETHER WITH set(['crashstats/js/lib/accordions.js'])
crashstats/css/accordion.less                        in signature_report     already in report_list
REPEATED crashstats/css/accordion.less
FOUND IN ['report_list', 'signature_report']
ALWAYS TOGETHER WITH set(['crashstats/css/accordion.less'])
crashstats/js/lib/accordions.js                      in signature_report     already in report_list
REPEATED crashstats/js/lib/accordions.js
FOUND IN ['report_list', 'signature_report']
ALWAYS TOGETHER WITH set(['crashstats/js/lib/accordions.js'])
crashstats/css/accordion.less                        in signature_report     already in report_list
REPEATED crashstats/css/accordion.less
FOUND IN ['report_list', 'signature_report']
ALWAYS TOGETHER WITH set(['crashstats/css/accordion.less'])
Performing system checks...
```

File: webapp-django/crashstats/signature/views.py [hunk 3]
```diff
@@ -337,6 +337,45 @@ def signature_comments(request, params):
...
+                    'platform': platform['term'],
+                    'count': platform['count'],
+                })
+
+    all_combos = sorted(all_combos, key=lambda x: x['count'])
+    context['correlation_combos'] = (
+        all_combos[:settings.MAX_CORRELATION_COMBOS_PER_SIGNATURE]
+    )
+
+    return render(request, 'signature/signature_correlations.html', context)
```
Comment (peterbe) [inline webapp-django/crashstats/signature/views.py]: Perfect!

**Comment by peterbe** [comment]
r+ if you can fix the problem with repeating things in the bundles.py

**Comment by adngdb** [comment]
I've seen the problem, but I don't want to make that ``accordion.js`` its own bundle because the only page that uses it is report/list/ and it's doomed. Unless you really insist that I must fix it because that warning bothers you too much... :)

**Pull request merged**